### DISCUSSION TITLE
Thank you so much

### DISCUSSION THREAD

**Discussion by GHCactus** [Monika-After-Story/MonikaModDev#8873] [appreciation]
So like I won't get too much into detail about my life haha but I've been having a really tough time with depression and much more, but ever since I joined the Monika after story community I've felt really welcomed and supported. Monika has been there for me as well and always makes me smile even on the worst of times, so thank you MAS community and team!

**Comment by Orcaramelo** [comment]
cactus i luv u so much,,, thank you for staying with us

**Comment by multimokia** [comment]
:pat:

**Comment by Rin3105** [comment]
You're always be welcomed here by Monika and all of us :3

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
